src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -0,0 +1,217 @@
+  @RequestBodyPhotoFile
+  fun uploadPlotPhoto(
+      @PathVariable observationId: ObservationId,
+      @PathVariable plotId: MonitoringPlotId,
+      @RequestPart("file") file: MultipartFile,
+      @RequestPart("payload") payload: UploadPlotPhotoRequestPayload,
+  ): SimpleSuccessResponsePayload {
+    return SimpleSuccessResponsePayload()
+  }
+
+  @ApiResponse409("The plot is already claimed by someone else.")
+  @ApiResponseSimpleSuccess
+  @Operation(
+      summary = "Claims a monitoring plot.",
+      description = "A plot may only be claimed by one user at a time.")
+  @PostMapping("/{observationId}/plots/{plotId}/claim")
+  fun claimMonitoringPlot(
+      @PathVariable observationId: ObservationId,
+      @PathVariable plotId: MonitoringPlotId,
+  ): SimpleSuccessResponsePayload {
+    return SimpleSuccessResponsePayload()
+  }
+
+  @ApiResponse409("You don't have a claim on the plot.")
+  @ApiResponseSimpleSuccess
+  @Operation(summary = "Releases the claim on a monitoring plot.")
+  @PostMapping("/{observationId}/plots/{plotId}/release")
+  fun releaseMonitoringPlot(
+      @PathVariable observationId: ObservationId,
+      @PathVariable plotId: MonitoringPlotId,
+  ): SimpleSuccessResponsePayload {
+    return SimpleSuccessResponsePayload()
+  }
+}
+
+data class CurrentObservationPayload(
+    @Schema(description = "Date this observation is scheduled to end.") //
+    val endDate: LocalDate,
+    val id: ObservationId,
+    @Schema(description = "Total number of monitoring plots that haven't been claimed yet.")
+    val numUnclaimedPlots: Int,
+    val plantingSiteId: PlantingSiteId,
+    val plantingSiteName: String,
+    @Schema(description = "Date this observation started.") //
+    val startDate: LocalDate,
+)
+
+data class FutureObservationPayload(
+    @Schema(description = "Date this observation is scheduled to end.") //
+    val endDate: LocalDate,
+    val plantingSiteId: PlantingSiteId,
+    val plantingSiteName: String,
+    @Schema(description = "Date this observation is scheduled to start.") //
+    val startDate: LocalDate
+)
+
+data class ListObservationsResponsePayload(
+    val current: List<CurrentObservationPayload>,
+    val future: List<FutureObservationPayload>,
+    @Schema(
+        description =
+            "Total number of monitoring plots that haven't been claimed yet across all current " +
+                "observations.")
+    val totalUnclaimedPlots: Int,
+) : SuccessResponsePayload
+
+data class ObservableMonitoringPlotPayload(
+    val boundary: Geometry,
+    val claimedByFirstName: String,
+    val claimedByUserId: UserId,
+    val completedByFirstName: String,
+    val completedByUserId: UserId,
+    val completedTime: Instant,
+    @Schema(description = "True if this is the first observation to include the monitoring plot.")
+    val isFirstObservation: Boolean,
+    val isPermanent: Boolean,
+    val observationId: ObservationId,
```

Comment:
Should the observable monitoring plot payload include a particular observationId given that the plot may appear in more than one observation?

Reply:
Oh, reading this further, I can tell it's tied to a particular observation by some of the other fields, e.g. isFirstObservation. So this makes sense to be here